src/style.css

```
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:not(:nth-child(n+5)) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars__star:not(:nth-child(n+4)) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:not(:nth-child(n+3)) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:not(:nth-child(n+3)),
.stars--3 .stars__star:not(:nth-child(n+4)),
.stars--4 .stars__star:not(:nth-child(n+5)),
.stars--5 .stars__star {
  background-image: url(images/star-active.svg);
}
```
It would be better to combine them together, to avoid duplicating setting the same property each time